### PR TITLE
Create superdomains.md

### PR DESCRIPTION
**Pull request by CHR15-**
Add base description and FAQs for Superdomains

### CODE REVIEW

File: wiki/superdomains.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+**Monetization**: How we monetize our users and channels, measured by Average Revenue Per User. 
+
+### How is work created for a Superdomain?
+
+At the Dropssite, we have ideation and prioritisation sessions with the entire team based on hitting a goal that we wish to achieve 
+as a company. Once we have an agreed list for each Superdomain, we enter the quarter with a backlog of work to complete.
+
+### How does this look week by week?
+
+Superdomain 'kickoff' sessions will be help every week on Tuesday (UA/M) and Wednesday (PS). In those sessions, we discuss the following:
```
Comment (szmraam) [inline wiki/superdomains.md:28]: Should this be "...will be held..." ?

Comment (CHR15-) [reply wiki/superdomains.md]: yep, ty!

File: wiki/superdomains.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+### How does this look week by week?
+
+Superdomain 'kickoff' sessions will be held every week on Tuesday (UA/M) and Wednesday (PS). In those sessions, we discuss the following:
+- The priority projects to be achieved that week, brought to the meeting by the project owner so everyone in the team has an understanding of what that project is and what it achieves
+- Deciding on the next steps to complete upcoming projects (organising meetings, resources required)
+- Deciding on the next steps to complete currently running projects (organising meetings, resources required)
+- Rough estimations from the involved on how long the selected projects will take for the week if appropriate
+
+Remember, the kickoff session is not suited to ideation. feedback on the upcoming projects can be given but ideally decisions on project changes shouldn't be made in these meetings (they should be noted and taken into project planning meetings run by the project owner).
```
Comment (thomasengebrand) [inline wiki/superdomains.md:34]: My point of view on the ideation is this: I don't think there needs to be any requirement to ideate in these calls. However if it makes sense for the project, we have the time and everyone has been brought up to speed on the background/goal of the project then I don't see any harm in ideating in these sessions. When we ideate I believe it's important that we are transparent and clearly show that all ideas voiced have been heard, captured and will be considered. I do agree we should not commit to in the meeting to which ideas we'll move forward with before exploring them in a bit more detail in a separate call as mentioned.

Comment (CHR15-) [reply wiki/superdomains.md:34]: agreed!!

**Pull request merged**